sdcm/mgmt/common.py

```diff
@@ -24,6 +26,53 @@ def duration_to_timedelta(duration_string):
     return datetime.timedelta(seconds=total_seconds)
 
 
+def get_manager_repo_from_defaults(manager_branch_name, distro):
+    with open("defaults/manager_versions.yaml", 'r') as mgmt_config:
+        manager_repos_by_branch_dict = yaml.load(mgmt_config, Loader=yaml.BaseLoader)["manager_repos_by_branch"]
+
+    branch_specific_repos = manager_repos_by_branch_dict.get(manager_branch_name, None)
+    assert branch_specific_repos, f"Couldn't find manager branch {manager_branch_name} in manager defaults"
+
+    known_distro_dict = {Distro.CENTOS7: "centos7",
```

Comment:
it could be in `Enum` object, instead of a definition inside the function... or either use the very same enum of `Distro.x`